@navikt/ds-react/layouts/common/utils.ts

```diff
@@ -0,0 +1,11 @@
+export const whiteDecoratorUtils = (toggle: boolean) => {
```

Comment:
Denne forstår jeg ikke helt. Kan du forklare?

Reply:
Når brødsmulestien og språkvelgeren tas i bruk i dekoratøren, så får body en ny div under dekoratøren med class-name decorator-utils-container. Om vi ønsker at tittelen skal gå i ett med dekoratøren, så må dette feltet få hvit bakgrunn. Se screenshot: 

<img width="1424" alt="Screenshot 2021-03-10 at 09 16 17" src="https://user-images.githubusercontent.com/6075402/110597861-61697e80-8181-11eb-95f0-ddd16de17289.png">

Mulig det er en mer elegant løsning på problemet, men jeg kom ikke på noe :/

Reply:
Nei, egentlig ikke, fordi mange apper bruker ønsker grå bakgrunn, mens andre ønsker hvit :/ så appene må på en eller annen måte definere hva den skal være

Reply:
Hvordan er APIet til dekoratøren nå? Kan det eventuelt sendes som en prop, eller kanskje man kan eksponere en funksjon?

Reply:
Hm, når jeg tenker meg om kan du være inne på noe. Vi kan legge inn en prop som kan endres med nav-dekoratoren-moduler client-side vha https://github.com/navikt/nav-dekoratoren-moduler.

Da kunne vi styrt bakgrunnsfargen via noe sånt:
```
import { setParams } from '@navikt/nav-dekoratoren-moduler'
setParams({
  utilsBackground: "white";
})
```
Men da får designsystemet en avhengighet til @navikt/nav-dekoratoren-moduler i komponentene, jeg vet ikke om det er ønskelig?

Reply:
Linja `document.getElementsByClassName("decorator-utils-container").item(0);` gir oss uansett en avhengighet til dekoratøren, så jeg tenker det bare er positivt å gjøre den avhengigheten eksplisitt.

Reply:
Ja, sant :) Er det mulig for oss å gjøre @navikt/nav-dekoratoren-moduler til en avhengighet kun for ProductPageLayout, eller må den være en peerAvhengighet til ds-react, for det er kanskje litt dumt?

Reply:
Per i dag håndteres avhengigheter globalt i ds-react, men vi kan endre det så det gjøres per komponent. Kan en annen løsning være at dekoratøren holdes helt utenfor, og at det blir opp til teamet å velge riktig bakgrunnsfarge?

Reply:
En fordel ved å gjøre det på den måten er at det blir lettere å ta i bruk ProductPageLayout (og andre layouts?) på sider som ikke har dekoratør.

Reply:
Men denne avgjørelsen skal selvfølgelig dere i designsystemet få ta :) Jeg synes det er pro / cons for begge alternativene

Reply:
Eksplisitt avhengighet til dekoratøren er fint. Brekker alt om dekoratøren ikke er der, eller går det fint?

Reply:
Jeg tenker at det vil være best å holde ds-react og dekoratøren separat om mulig. Vil det være mulig å ha noe lignende i dekoratøren?: `document.getElementsByClassName('layout-bg-farge--white');` -> Endre klasse på dekroratør util

Reply:
@KenAJoh, er det ikke dumt å dra inn logikk i dekoratøren for å sette bakgrunnsfarge basert på en klasse i en enkelt app / layout? Da får man motsatt problem. Jeg tenker det er best om appene styrer hvordan dekoratøren skal oppføre seg.

Reply:
@andnorda  trur ikke det krasjer om dekoratøren ikke er der, men potensielt må man installere @navikt/nav-dekoratoren-moduler uten å ha dekoratør (om det er noe vi skal støtte)

Reply:
implementert med @navikt/nav-dekoratoren-moduler, lukker